packages/types/src/Metadata/v2/static.ts

```diff
@@ -0,0 +1,7 @@
69636573184d6f64756c652042616c616e636573030004000000013473726d6c5f62616c616e636573184d6f64756c651c53657373696f6e040004000000013073726d6c5f73657373696f6e184d6f64756c651c5374616b696e67050004000000013073726d6c5f7374616b696e67184d6f64756c652444656d6f6372616379060004000000013873726d6c5f64656d6f6372616379184d6f64756c651c436f756e63696c070004000000014c73726d6c5f636f756e63696c3a3a7365617473184d6f64756c6534436f756e63696c566f74696e67080004000000015073726d6c5f636f756e63696c3a3a766f74696e67184d6f64756c6538436f756e63696c4d6f74696f6e73090004000000015473726d6c5f636f756e63696c3a3a6d6f74696f6e73184d6f64756c651c4772616e6470610a0004000000013073726d6c5f6772616e647061184d6f64756c652054726561737572790b0004000000013473726d6c5f7472656173757279184d6f64756c6520436f6e74726163740c0004000000013473726d6c5f636f6e7472616374184d6f64756c65105375646f0d0004000000012473726d6c5f7375646f184d6f64756c65013873726d6c5f74696d657374616d70184d6f64756c650100013873726d6c5f636f6e73656e737573184d6f64756c650100013073726d6c5f696e6469636573184d6f64756c650100013473726d6c5f62616c616e636573184d6f64756c650100013073726d6c5f73657373696f6e184d6f64756c650100013073726d6c5f7374616b696e67184d6f64756c650100013873726d6c5f64656d6f6372616379184d6f64756c650100014c73726d6c5f636f756e63696c3a3a7365617473184d6f64756c650100015073726d6c5f636f756e63696c3a3a766f74696e67184d6f64756c650100015473726d6c5f636f756e63696c3a3a6d6f74696f6e73184d6f64756c650100013073726d6c5f6772616e647061184d6f64756c650100013473726d6c5f7472656173757279184d6f64756c650100013473726d6c5f636f6e7472616374184d6f64756c650100012473726d6c5f7375646f184d6f64756c650100013473725f7072696d697469766573244163636f756e74496401040000001b022c73726d6c5f73797374656d2c4576656e745265636f72640401306e6f64655f72756e74696d65144576656e74010801147068617365012c73726d6c5f73797374656d14506861736501146576656e7401306e6f64655f72756e74696d65144576656e74012c73726d6c5f73797374656d1450686173650208384170706c7945787472696e736963000004000000123046696e616c697a6174696f6e01000001306e6f64655f72756e74696d65144576656e7402381873797374656d000004000000012c73726d6c5f73797374656d144576656e741c696e6469636573010004000000023073726d6c5f696e6469636573205261774576656e7408013473725f7072696d697469766573244163636f756e744964122062616c616e636573020004000000023473726d6c5f62616c616e636573205261774576656e7408013473725f7072696d697469766573244163636f756e744964161c73657373696f6e030004000000023073726d6c5f73657373696f6e205261774576656e7404141c7374616b696e67040004000000023073726d6c5f7374616b696e67205261774576656e740816013473725f7072696d697469766573244163636f756e7449642464656d6f6372616379050004000000023873726d6c5f64656d6f6372616379205261774576656e740816013473725f7072696d697469766573244163636f756e7449641c636f756e63696c060004000000024c73726d6c5f636f756e63696c3a3a7365617473205261774576656e7404013473725f7072696d697469766573244163636f756e74496438636f756e63696c5f766f74696e67070004000000025073726d6c5f636f756e63696c3a3a766f74696e67205261774576656e74041b3c636f756e63696c5f6d6f74696f6e73080004000000025473726d6c5f636f756e63696c3a3a6d6f74
```

Comment:
I just built @xlc's branch on paritytech/substrate, and `state_getMetadata` gave me somthing starting with `0x1445`

Reply:
He did a rebase i think, will update it.